subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskParametersIntegrationTest.groovy

```diff
@@ -102,6 +102,7 @@ class TaskParametersIntegrationTest extends AbstractIntegrationSpec implements V
             task test(type: TaskWithTwoFileCollectionInputs) {
                 inputs1 = files("input1.txt", "input2.txt")
                 inputs2 = files("input3.txt")
+                output = layout.buildDirectory.dir("out").get().asFile
```

Comment:
```suggestion
                // pick a custom subdir to avoid flaky up-to-date status due to other stuff in `build` changing
                output = layout.buildDirectory.dir("out").get().asFile
```

Reply:
I don't think this needs a comment in the code, because the test itself does not care which output directory is used. The context in the commit/PR should be enough.